modin/core/execution/dask/implementations/pandas_on_dask/partitioning/partition.py

```diff
@@ -268,30 +273,6 @@ def ip(self):
             self._ip_cache = DaskWrapper.materialize(self._ip_cache)
         return self._ip_cache
 
-    def split(self, split_func, num_splits, *args):
-        """
-        Split the object wrapped by the partition into multiple partitions.
-
-        Parameters
-        ----------
-        split_func : Callable[pandas.DataFrame, List[Any]] -> List[pandas.DataFrame]
-            The function that will split this partition into multiple partitions. The list contains
-            pivots to split by, and will have the same dtype as the major column we are shuffling on.
-        num_splits : int
-            The number of resulting partitions (may be empty).
-        *args : List[Any]
-            Arguments to pass to ``split_func``.
-
-        Returns
-        -------
-        list
-            A list of partitions.
-        """
-        outputs = DaskWrapper.deploy(
-            split_func, [self._data] + list(args), num_returns=num_splits, pure=True
-        )
```

Comment:
what about the missing `pure=True` parameter in the base implementation, is it significant?

Reply:
`pure=True` is a default value for `DaskWrapper.deploy`, so no problem